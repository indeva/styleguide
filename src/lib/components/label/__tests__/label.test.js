import React from 'react';
import renderer from 'react-test-renderer';
import Label from '..';

describe('Label', () => {
  it('renders the label', () => {
    const component = renderer.create(<Label htmlFor="Test">Test</Label>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
