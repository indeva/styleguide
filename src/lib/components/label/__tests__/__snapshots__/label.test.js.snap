// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label with children render label properly 1`] = `
<label
  className="label"
  htmlFor={null}
>
  label
</label>
`;

exports[`Label with custom class render label properly 1`] = `
<label
  className="label my class"
  htmlFor={null}
>
  label
</label>
`;

exports[`Label with htmlFor render label properly 1`] = `
<label
  className="label"
  htmlFor="name"
>
  label
</label>
`;

exports[`Label with modifier disabled render label properly 1`] = `
<label
  className="label label--disabled"
  htmlFor={null}
>
  label
</label>
`;

exports[`Label with modifier featured render label properly 1`] = `
<label
  className="label label--featured"
  htmlFor={null}
>
  label
</label>
`;

exports[`Label with modifier small render label properly 1`] = `
<label
  className="label label--small"
  htmlFor={null}
>
  label
</label>
`;
